packages/utilities/src/focus.ts

```diff
@@ -196,11 +196,11 @@ export function isElementTabbable(element: HTMLElement): boolean {
 
   return (
     !!element && isFocusableAttribute !== 'false' &&
+    tabIndex >= 0 || !tabIndex &&
```

Comment:
should we just check if `!(tabIndex == -1)` for all other values its should be focussable